pkg/download/config_creation/config_creation_test.go

```diff
@@ -0,0 +1,71 @@
+/*
+ * @license
+ * Copyright 2025 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package config_creation_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/dynatrace/dynatrace-configuration-as-code/v2/pkg/download/config_creation"
+)
+
+type uidStruct struct {
+	Uid string `json:"uid"`
+}
+
+type idStruct struct {
+	Id string `json:"id"`
+}
+
+func TestCreateConfig(t *testing.T) {
```

Comment:
I would also add some additional error test cases. Specially the field that I have in mind is `structPointer` as declared as any. The specific test for `replaceParam`.